src/Components/Facility/ConsultationDetails/Events/GenericEvent.tsx

```diff
@@ -1,8 +1,10 @@
 import type { ReactNode } from "react";
+import consultationMessage from "../../../../../src/Locale/en/Consultation.json";
 interface IProps {
   values: Record<string, unknown>;
 }
 
+type ConsultationMessageKeys = keyof typeof consultationMessage;
```

Comment:
You shouldn't be importing the translations file directly. Use `useTranslation` hook to retrieve the display text instead. Ref: https://github.com/coronasafe/care_fe/blob/f76248c997c1153a9f859511a6678ada18dd0af9/src/Components/Facility/Consultations/DailyRoundsList.tsx#L24